### PR TITLE
Added bug issue template

### PR DESCRIPTION
**Pull request by kashimiz**
Added issue template for reporting bugs.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+***If JavaScript***
+- durable-functions npm module version:
+
+***If deployed to Azure***
+- Timeframe issue observed:
+- Function App name:
+- Function name(s):
+- Region:
+- Invocation IDs:
```
Comment (cgillum) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Invocation ID isn't particularly useful. I would mark this as optional. What we _really_ want is the orchestration execution ID.

Comment (kashimiz) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done; moved invocation ID field to 'additional info' section at bottom, changed 'orchestration instance ID' to 'orchestration execution IDs'.

Comment (cgillum) [reply .github/ISSUE_TEMPLATE/bug_report.md]: We actually want orchestration _instance_ IDs (which you had previously), not _execution_ IDs.

Comment (kashimiz) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I've opened #564 to address this.

**Pull request merged**